### ISSUE TITLE
free form objects not handled correctly

### ISSUE THREAD

**Issue by Jyhess** [zalando/connexion#781]
### Description

Connexion filter out all values of [free form object](https://swagger.io/docs/specification/data-models/data-types/#free-form).

swagger/swagger.yaml
```
openapi: "3.0.1"
info:
  title: "Simple example"
  version: "1"
servers:
- url: "http://localhost:8080/v1"
paths:
  /json_body:
    post:
      summary: "Simple POST with free form body"
      operationId: "controller.json_body"
      requestBody:
        content:
          application/json:
            schema:
              type: "object"
      responses:
        201:
          description: "OK"
```
controller.py
```
from aiohttp import web


async def json_body(body):  # noqa: E501
    print(body)
    return web.Response(status=201)
```

main.py
```
import connexion


def main():
    app = connexion.AioHttpApp(__name__, specification_dir='./swagger/')
    app.add_api('swagger.yaml', arguments={'title': 'Echo service'}, pass_context_arg_name='request')
    app.run(port=8080)


if __name__ == '__main__':
    main()
```

### Expected behaviour

When I call ```curl http://localhost:8080/v1/json_body -d '{"foo": "bar"}' -H "Content-Type: application/json"```
_{"foo": "bar"}_ is displayed in logs.

### Actual behaviour

_{}_ is displayed in logs.

### Additional info:

python 3.6.6
connexion 2.0.1
aiohttp 3.4.4

**Comment by dtkav** [comment]
Thanks for the ticket @Jyhess - I'll try to take a look at the end of the week.

**Comment by dtkav** [comment]
potential fix: #789

**Comment by Jyhess** [comment]
> potential fix: #789

I confirm this PR fix this issue.

**Comment by dtkav** [comment]
Closing this ticket, as the fix has been merged and will be part of the next release.
Thanks @Jyhess for filing the issue, and @krise3k for the fix!
